pkgs/by-name/va/vault-unseal/package.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "vault-unseal";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lrstanley";
+    repo = "vault-unseal";
+    rev = "v${version}";
+    hash = "sha256-vjU4080uCId/73F7CJKDtk9b1siCPIZOaSczKMNf0LE=";
+  };
```

```suggestion
let
	version = "0.5.1";
in
buildGoModule {
  pname = "vault-unseal";

  src = fetchFromGitHub {
    owner = "lrstanley";
    repo = "vault-unseal";
    rev = "v${version}";
    hash = "sha256-vjU4080uCId/73F7CJKDtk9b1siCPIZOaSczKMNf0LE=";
  };
```

Can we drop the `rec`?

---

I guess so but we will still need an `inherit version;` in the `buildGoModule`, as it still needs to be there.

---

Yep, right you are